eemeter/models/temperature_sensitivity.py

```diff
@@ -32,6 +39,21 @@ class BaseloadHeatingCoolingModelParameterType(ParameterType):
         "cooling_slope"
     ]
 
+
+class BaseloadSeasonalHeatingCoolingModelParameterType(ParameterType):
+    # TODO: Should we have separate input and results parameter types?
```

Comment:
If there are inputs that aren't ranges or initial starting values for parameters that get fit, I'm thinking we should deal with them a different way (e.g. as kwargs).